workflows/sock-shop-promProbe/sock-shop.chartserviceversion.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  createdAt: 2021-05-24T10:28:08Z
+  name: sock-shop-promProbe
+  version: 0.1.0
+  annotations:
+    categories: sock-shop
+    chartDescription: Injects chaos on sock-shop application using prom probe.
+spec:
+  displayName: Sock-Shop-Chaos
+  categoryDescription: >
+    It install sock-shop application, generate loads, inject chaos on sock-shop,
+    uninstall the application and revert the chaos
+  experiments:
+    - pod-delete
+    - pod-cpu-hog
+    - pod-memory-hog
+    - pod-network-loss
+    - disk-fill
+  keywords:
+    - Kubernetes
+    - Sock-Shop
+    - Pod
+    - Memory
+    - CPU
+    - Network
+    - Disk
+  platforms:
+    - GKE
+    - Minikube
+    - Packet(Kubeadm)
+    - EKS
+    - AKS
+  maintainers:
+    - name: oumkale
+      email: oumkale@chaosnative.com
+  provider:
+    name: ChaosNative
+  links:
+    - name: Kubernetes Slack
+      url: https://slack.kubernetes.io/
+    - name: Documentation
+      url: https://docs.litmuschaos.io/docs/chaoshub/#generic-chaos
+  icon:
+    - url:
```

Comment:
Since we have image here(workflows/icons/sock-shop-promProbe.png), will you please update here link? Same for cmd probe.